### ISSUE TITLE
Plugin does not check if database already exists, tries to create (which requries admin privs)

### ISSUE THREAD

**Issue by m00dawg** [agrif/OctoPrint-InfluxDB#4]
I'm running into an issue where the plugin keeps wanting to create the database even though it already exists:

```
2019-11-29 20:02:57,366 - octoprint.plugins.influxdb - ERROR - Cannot create InfluxDB database.
Traceback (most recent call last):
  File "/home/pi/oprint/local/lib/python2.7/site-packages/octoprint_influxdb/__init__.py", line 67, in influx_try_connect
    db.create_database(dbname)
  File "/home/pi/oprint/local/lib/python2.7/site-packages/influxdb/client.py", line 626, in create_database
    method="POST")
  File "/home/pi/oprint/local/lib/python2.7/site-packages/influxdb/client.py", line 450, in query
    expected_response_code=expected_response_code
  File "/home/pi/oprint/local/lib/python2.7/site-packages/influxdb/client.py", line 302, in request
    raise InfluxDBClientError(response.content, response.status_code)
InfluxDBClientError: 403: {"error":"error authorizing query: octoprint not authorized to execute statement 'CREATE DATABASE OctoPrint', requires admin privilege"}
```

The database, however, exists:

```
> show databases;
name: databases
name
----
_internal
...
OctoPrint
```

If I give Influx user admin rights, it works, but that's definitely not ideal (I'd prefer to give it read/write only to the database).

**Comment by agrif** [comment]
Aha! thanks for the report. I've just released [1.0.1](https://github.com/agrif/OctoPrint-InfluxDB/releases/tag/1.0.1) which checks to see if the database exists before trying to create it.

**Comment by m00dawg** [comment]
Just providing quick feedback, yep that fixed it for me, thanks! :guitar:
